templates/default/mongod.conf.erb

```diff
@@ -0,0 +1,18 @@
+# This file was generated by Chef Infra
+# Do NOT modify this file by hand.
+net:
+  port: <%= @port %>
+  bindIp: <%= @bind_ip %>
+  maxIncomingConnections: <%= @max_connections %>
+processManagement:
+  fork: true
+  pidFilePath: /var/run/mongodb/mongod.pid
+  timeZoneInfo: /usr/share/zoneinfo
+storage:
+  dbPath: <%= @data_dir %>
+  journal:
+    enabled: true
+systemLog:
+  destination: <%= @log_dest %>
+  logAppend: true
+  path: <%= @log_path %>
```

Comment:
If `@log_dest` is set to `syslog`, this can't be set.

Reply:
Yeah I changed that yesterday and forgot to push :)

Reply:
I'll go back and push the changes when I get in tomorrow! (I was too excited that all the tests were passing haha...)